frontend/assets/stylesheets/components/_articles.scss

```diff
@@ -244,12 +244,19 @@
     float: left;
     padding-right: $gs-gutter/3;
 
-    @include mq(tablet) {
+    @include mq(wide) {
         @include fs-header(4);
         line-height: get-line-height(header, 2);
```

Comment:
Might be worth just explicitly setting font size here and using the font-only mixin?